flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/validation/DefaultValidator.java

```diff
@@ -178,7 +178,7 @@ private Optional<String> validateJobSpec(
             return Optional.empty();
         }
 
-        if (job.getJarURI() == null) {
+        if (job.getJarURI() == null || job.getJarURI().equals("")) {
```

Comment:
This condition in this `if` check could be replaced by `StringUtils.isEmpty(job.getJarURI())`

Reply:
`isEmpty` not found, using `isNullOrWhitespaceOnly` instead